lib/src/main/java/com/auth0/jwt/algorithms/Algorithm.java

```diff
@@ -182,8 +182,10 @@ public static Algorithm HMAC256(byte[] secret) throws IllegalArgumentException {
      *
      * @param keyProvider the provider of the Public Key and Private Key for the verify and signing instance.
      * @return a valid ECDSA256 Algorithm.
+     * @deprecated This method is deprecated since Java 15+ disables SECP-256K1 Curve. Using this in Java 15+ will throw a SignatureException
```

Comment:
How about:

"The SECP-256K1 Curve algorithm has been disabled beginning in Java 15, and removed in Java 16+. Use of this method in those unsupported Java versions will throw a {@link SignatureException}. This method will be removed in the next major version. See <a href="https://bugs.openjdk.java.net/browse/JDK-8251547"></a> for additional information"